docs/user/metacat/source/configuration.rst

```diff
@@ -28,99 +25,59 @@ You can always open the configuration screen from within Metacat by typing::
 
   http://<your_context_url>/admin
 
-Initial Configuration
----------------------
-Before you can log in to the Metacat and configure it, you are required to 
-confirm Metacat's back-up location and authentication configuration (if not 
-already configured). Metacat will automatically attempt to locate an existing 
-back-up directory, but you may need to correct the value or specify a directory 
-(if the installation is new, or if Metacat was unable to determine the location 
-of an existing back-up directory). The authentication configuration is required 
-for logging in to the Metacat and for defining administrative accounts. 
-Instructions for `Changing Authentication Configuration without Authentication`_ 
-are included at the end of this section.
-
-Back-up Configuration
-~~~~~~~~~~~~~~~~~~~~~
-To preserve its configuration settings, Metacat backs up crucial configuration details to 
-a directory outside the application directories. Because a new installation/upgrade 
-does not know where this external directory is, Metacat uses a discovery 
-algorithm to locate it. If Metacat cannot identify a backup directory, you will 
-see the Backup Directory Configuration screen.
-
-.. figure:: images/screenshots/image011.png
-   :align: center
-
-   Configuring the Backup Directory.
```

Comment:
I believe we still have the backup directory functionality? For a fresh install, you should still see the backup config page, before you see the admin setup?

Reply:
I have added this back in, with an updated screenshot and wording/formatting. Note - I had to remove my entire `/var/metacat` folder in order to trigger the backup config setting screen.